src/appkit.cs

```diff
@@ -904,6 +904,22 @@ interface NSApplicationDelegate {
 		void UserDidAcceptCloudKitShare (NSApplication application, CKShareMetadata metadata);
 #endif
 	}
+
+	[Mac (10, 12, 1)]
+	[Category]
+	[BaseType (typeof(NSApplication))]
+	interface NSApplication_NSTouchBarCustomization
+	{
+		[Export ("isAutomaticCustomizeTouchBarMenuItemEnabled")]
+		bool GetAutomaticCustomizeTouchBarMenuItemEnabled ();
```

Comment:
I mentioned it inside another comment but **don't inline** unless it's confirmed it won't be throwing inside the debugger, e.g. on earlier macOS or when a touchbar is not available...